content/tooling/wrangler/commands.md

```diff
@@ -152,6 +153,23 @@ If you would like to be able to publish your code to multiple places, please see
 
   Additionally, you can preview different environments. This is useful if you have different builds for different environments (like staging vs. production), but typically isn't needed. For more information see the [environments documentation](/tooling/wrangler/configuration/environments).
 
+#### Making preview work with WSL2
+
+##### Setting $BROWSER to your browser binary
+  By default, `wrangler` sees WSL/2 as a linux environment and invokes `xdg-open` to open the browser. To make it work with WSL/2, you should set your `$BROWSER` to your browser binary.
+
+  eg. `$ export BROWSER='/mnt/c/tools/firefox.exe'
+  `
+  Spaces in filepath are not common in linux, however it's not illegal and needs to be quoted. Some programs like `xdg-open` breaks on [paths with spaces](https://github.com/microsoft/WSL/issues/3632#issuecomment-432821522), so you would need a workaround like linking the binary to your `/usr/local/bin`.
+
+  eg. $ ln -s '/mnt/c/Program Files/Mozilla Firefox/firefox.exe' firefox
+      $ export BROWSER=firefox
+
+##### Setting $BROWSER to `wsl-open`
+  Another option is to install [wsl-open](https://github.com/4U6U57/wsl-open#standalone) and set the $BROWSER env variable to `wsl-open`, via `wsl-open -w` (which associate wsl-open with links)
+
+  If you're using WSL2, you would want to install via [standalone method](https://github.com/4U6U57/wsl-open#standalone), as npm is on version 1.3.0 which doesn't support WSL2 yet.
```

Comment:
```suggestion
  If you're using WSL 2, you will need to install `wsl-open` via their [standalone method](https://github.com/4U6U57/wsl-open#standalone) rather than through `npm`. This is because their npm package has not yet been updated with WSL 2 support.
```